app/connector/BindingTariffClassificationConnector.scala

```diff
@@ -73,6 +73,20 @@ class BindingTariffClassificationConnector @Inject() (
     s"${appConfig.bindingTariffClassificationUrl}/cases?$queryString"
   }
 
+  private def buildQueryAllQueuesURL(
```

Comment:
Is this function necessary?

If we change `buildQueryUrl` to accept a sequence of queueId's then it's the same function, since `queueIds.mkString(",")` for a single queueId will just return the queueId itself.

Reply:
you're right, thanks! We did as you suggested, removed the unnecessary method and just kept one that accepts a sequence of queueIds